### DISCUSSION TITLE
支援自訂 plugin repo 來自動檢查更新語言包

### DISCUSSION THREAD

**Discussion by bluelovers** [bluelovers/idea-l10n-zht#25] [Announcements]
![image](https://user-images.githubusercontent.com/167966/171050664-1b79bc8c-1984-4bb5-b7fb-c3d2a5aaf745.png)

`https://github.com/bluelovers/idea-l10n-zht/raw/master/plugin-dev-out/updatePlugins.xml`

![image](https://user-images.githubusercontent.com/167966/171050566-570d3fc3-de4e-4dc5-a17a-c2a2d811a695.png)

![image](https://user-images.githubusercontent.com/167966/171050682-11c9d3d5-48cf-4332-9410-cad40778c416.png)

**Comment by bluelovers** [comment]
@SiongSng @benebsiny @ruyut @Marniew @hwhsu1231 @creamgod45

**Reply by SiongSng** [reply]
:+1:

**Comment by Marniew** [comment]
🤩
👍

**Comment by bluelovers** [comment]
@hwhsu1231 更新支援在`未安裝過`的狀態下能用名字搜尋到這個語言包

![image](https://user-images.githubusercontent.com/167966/203631133-44800569-a673-482f-a4e5-8334a43cbbd0.png)

https://github.com/bluelovers/idea-l10n-zht/commit/bfdbcd072833a61b0ed506d7f73277950bef8843

**Reply by hwhsu1231** [reply]
太好了，這樣對於使用者來說就更方便安裝了！！

**Reply by SiongSng** [reply]
太棒了

**Comment by bluelovers** [comment]
https://github.com/bluelovers/idea-l10n-zht/issues/32#issuecomment-1319031648

https://github.com/bluelovers/idea-l10n-zht/issues/26#issuecomment-1145329040

https://github.com/bluelovers/idea-l10n-zht/issues/28#issuecomment-1163539936
